pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -94,7 +94,7 @@ def prepare_bokchoy_run(self):
             # Reset the database
 
             # Collect static assets
-            sh("paver update_assets --settings=bok_choy")
+            sh("paver update_assets --settings=bok_choy_static")
```

Comment:
It'd probably be worthwhile to also throw in a comment here highlighting that the settings.py file used to collect static assets is a different one. Otherwise glancing through the code it might be hard to visually parse that out if you don't already know.